### DISCUSSION TITLE
Hello. Could you tell me in which file I can check if LLC belongs to the virtual index and virtual tag structure？

### DISCUSSION THREAD

**Discussion by szuhzw** [ChampSim/ChampSim#559] [Q&A]
Hello. Could you tell me in which file I can check if LLC belongs to the virtual index and virtual tag structure？

**Comment by szuhzw** [comment]
@maccoymerrell @ngober @sethpugsley @elbs @vinsanity101

**Comment by ngober** [comment]
Have you tried looking anywhere already that might answer your question? In particular, the function named [`try_hit()`](https://github.com/ChampSim/ChampSim/blob/aa336e838027563196eeb721be027f48d63a1b39/src/cache.cc#L167) in a file named "cache.cc"? That seems like it could be relevant to your question.
